test.js

```diff
@@ -1,7 +1,72 @@
 import test from 'ava';
 import m from '.';
 
-test(t => {
-	t.is(m('5.1.2600'), 'XP');
-	t.is(m('10.0.10240'), '10');
+// Test using versions strings sourced from https://www.gaijin.at/en/lstwinver.php
```

Comment:
This would benefit from having tests testing actual server instances of windows as well